edx/front-end-cookie-cutter-application
# build(docker): add initial docker configuration

Goal is to start the dev server from a Docker container

To test:
* `docker-compose build`
* `docker-compose up`
* Should start dev server with hot-reloading
